### PR TITLE
New PR Ship to store

### LINKED ISSUE #105: Ship to store
If an item is not available for same-day pick-up but should be available for pick-up when inventory becomes available at the pick-up location, HotWax should allow BOPIS orders. When imported, OMS processes them as regular shipping orders and routes BOPIS items to a fulfillment location with available inventory to ship them to the customer's selected pick-up location. When orders arrive at the store, store associates can modify the order status in OMS to "scheduled" and notify customers that their orders are ready for pick-up.

### PR DESCRIPTION
**Pull request by Dhiraj1405**
closes #105

### CODE REVIEW

**Comment by Shreyaakothari** [comment]
closes #105

File: bopis/fulfillment/ship-to-store.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Allowing customers to place BOPIS orders without inventory constraint eliminates the frustration of constantly checking for same-day pickup availability. The OMS will process these orders as regular shipping orders and intelligently route the order to the pickup location. Once the order arrives at the store, store associates will change the status of orders in OMS to “scheduled” and notify customers to pick up their orders.
+
+Retailers using Hotwax Commerce can change their frontend experience for products which are unavailable for same day pick-up from 'Out of Stock' to 'Ship to store’ or ‘Pick up in two days’. In the backend, the checkInventory API will fetch all available BOPIS locations and for locations where product inventory is out of stock the PDP will display a pickup later option.
+
+
+### FAQs
+
+Q1: What will the product detail page (PDP) display when the customer searches for all available BOPIS locations and some pickup locations have inventory while others do not?
```
Comment (dt2patel) [inline bopis/fulfillment/ship-to-store.md]: Look at previous PR FAQ for question formatting and update these to match.

**Review by dt2patel — CHANGES_REQUESTED** [review]
Need to update question formatting.

File: bopis/fulfillment/ship-to-store.md (hunk 1 above)
Comment (Dhiraj1405) [reply bopis/fulfillment/ship-to-store.md]: Updated

**Review by dt2patel — APPROVED** [review]

**Pull request merged**